admin_guide/managing_networking.adoc

```diff
@@ -275,6 +275,15 @@ above example, suppose `www.foo.com` resolved to `10.11.12.13` and has a DNS TTL
 of one minute, but was later changed to `20.21.22.23`. {product-title} will then
 take up to one minute to adapt to these changes.
 
+[NOTE]
+====
+The default installer always allows pods access to the external interface of the 
```

Comment:
@JacobTanenbaum context is not clear to me. It seems that you really just need access to the DNS server's IP address? Is this the installer setting up a firewall rule to allow the pod to access the external interface? Or is it the DENY rule must be last? What is it that we care about in this discussion?